### PR TITLE
Adding support for icons in place of the center softkey text

### PR DESCRIPTION
**Pull request by AdrianMachado**
Finishing up the softkey work by allowing developers to build components with icons as the center softkey as listed in the kaios design spec

Screen Shot 2019-09-08 at 11 46 47 PM

image

Addresses #2

### CODE REVIEW

File: src/components/SoftKey/SoftKey.jsx [hunk 1]
```diff
@@ -33,66 +31,61 @@ const Button = props => {
       onClick={handleButtonClick}
       onFocus={handleCheckFocus}
     >
-      {props.icon ? <i class={props.icon} /> : null}
+      {props.icon ? <span className={props.icon} /> : null}
```
Comment (Phidelux) [inline src/components/SoftKey/SoftKey.jsx]: This seems to be the only relevant change. We shouldn't mix styling changes with fixes or new features, especially not if this changes are not consistent with the existing code base.

File: src/components/SoftKey/SoftKey.jsx [hunk 2]
```diff
@@ -6,15 +6,13 @@ import './SoftKey.scss';
 const prefixCls = 'kai-softkey';
 
 const Button = props => {
-  const {
-    handleClick
-  } = props;
+  const { handleClick } = props;
```
Comment (Phidelux) [inline src/components/SoftKey/SoftKey.jsx]: Why are `props.icon` and `props.text` not included here?

Comment (AdrianMachado) [reply src/components/SoftKey/SoftKey.jsx]: Good catch

File: src/components/SoftKey/SoftKey.jsx (hunk 1 above)
Comment (AdrianMachado) [reply src/components/SoftKey/SoftKey.jsx]: My bad, linter did it automatically

**Pull request merged**